app/uk/gov/ons/sbr/models/units/EnterpriseUnit.scala

```diff
@@ -5,7 +5,7 @@ import play.api.libs.json.{ Json, OFormat }
 import uk.gov.ons.sbr.models.DataUnit
 
 case class EnterpriseUnit(
-  @ApiModelProperty(example = "", dataType = "java.lang.Long") id: String,
+  @ApiModelProperty(value = "Unit identifier", required = true, hidden = false, example = "", dataType = "java.lang.Long") id: String,
```

Comment:
Add an example

Reply:
Sure - but I've just moved the existing annotation from where it was invalid ...